### ISSUE TITLE
licence type ?

### ISSUE THREAD

**Issue by MatPain** [Slluxx/fivem-servermanager#3]
Hello,

thanks you for your great server manager.

I did a commit one week ago but no answer from you.

Its it possible to modify your code and publish to public it ?

thanks you

**Comment by Slluxx** [comment]
No. You want to clean the cache in restart, which you dont need. The cache is updating itself every time the the server is restarted.

You can modify your local version oft the script but not release it yourself.

**Comment by MatPain** [comment]
updating ? but not clean for developing

**Comment by Slluxx** [comment]
The manager is for production and endusers, not for developing. The cache is updating itself so its not needed to delete it. That actually slows the serverstartup which is not efficient.
